openshift_tools/cloud/aws/ebs_snapshotter.py

```diff
@@ -151,6 +154,7 @@ def create_snapshots(self, schedule, script_name=None, dry_run=False, sleep_betw
             # Status: permanently disabled
             # pylint: disable=broad-except
             except Exception as ex:
+                # pylint: disable=no-member
```

Comment:
Please add a reason block here per our best practices:
https://github.com/openshift/openshift-tools/blob/prod/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code